service/hook/parser/parse.go

```diff
@@ -232,9 +232,13 @@ func (p *parser) Parse(req *http.Request, secretFunc func(string) string) (*core
 		}
 		return hook, repo, nil
 	case *scm.PullRequestHook:
-		if v.Action != scm.ActionOpen && v.Action != scm.ActionSync {
+		switch v.Action {
+		case scm.ActionOpen, scm.ActionClose, scm.ActionSync:
```

Comment:
Should this also include `scm.ActionMerge` to capture PR merge events?

Reply:
I'm not the most knowledgeable when it comes to Go and this codebase is new for me, but I've been digging through the webhook system. Here is the piece that I'm wondering about: https://github.com/drone/go-scm/blob/master/scm/driver/bitbucket/webhook.go#L49